### ISSUE TITLE
REQUEST: Toggle Max_Stack Keybinding

### ISSUE THREAD

**Issue by chaibronz** [conformal/spectrwm#434]
Please add key binding to toggle 'Max_Stack' on/off.  Max_Stack acts as a 'monocle' layout but is only enabled by cycling through other layouts:  ie. Vertical_Stack --> Horizontal_Stack --> Max_Stack. 

Other window modes such as "floating", "fullscreen" and "maximize" can be toggled on/off. I think it would make sense to re-purpose the "maximize" toggle to function the same as the "Max_stack" layout. 

Key differences:

"Maximize" - removes gaps and doesn't allow you to cycle through other windows on the workspace (strictly maximizes current window only and until maximize is toggled off again)

"Max Stack" - maximizes window while maintaining gaps and lets you cycle through other open windows on the workspace

In a nutshell, please re-purpose 'Maximize_Toggle' to function as 'Max_Stack', or add keybinding to toggle on/off 'Max_Stack'.

EDIT: I did find there is a "layout_max" binding available which goes straight into the MAX_Stack layout. It doesn't toggle back out, but it gets you in lol  -- Anyway to define these "layout" bindings to toggle in then back out to the prior layout setting?

**Comment by LordReg** [comment]
`prior_layout` (needs to be bound) can be used to toggle the layout.
